src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -189,6 +189,18 @@ Environment variable: `$WEB_EXT_BROWSER_CONSOLE=true`
 
 Note: The browser console may not show all debugging output from content-scripts. Use the web console when debugging content-scripts.
 
+
+#### `--devtools`
+
+This opens the Developer Tools for the installed extension on startup. See [this documentation](https://extensionworkshop.com/documentation/develop/debugging/) for more information. Example:
+
+```shell
+web-ext run --devtools
+```
+
+Note: The opened Developer Tools may not show all debugging output from content-scripts. Use the web console when debugging content-scripts.
+Note #2: This option only works with Firefox 106 and newer.
```

Comment:
We might want to add the web-ext version as well

Reply:
Thanks for the suggestion!